plugins/scaffolder/src/components/fields/EntityPicker/EntityPicker.tsx

```diff
@@ -21,22 +21,53 @@ import {
 import { TextField } from '@material-ui/core';
 import FormControl from '@material-ui/core/FormControl';
 import Autocomplete from '@material-ui/lab/Autocomplete';
+import { JSONSchema7 } from 'json-schema';
 import React, { useCallback, useEffect } from 'react';
 import useAsync from 'react-use/lib/useAsync';
+import { z } from 'zod';
+import zodToJsonSchema from 'zod-to-json-schema';
 import { FieldExtensionComponentProps } from '../../../extensions';
 
+/**
+ * @public
+ */
+export const EntityPickerInput = z.object({
```

Comment:
I'm wondering if it makes sense to move these `zod` definitions out and have a standard that we have a `schema.ts` in each of the `FieldExtension` component folders?

Reply:
And it would export `EntityPickerSchema` with the jsonschema stuff done in there. 

Or we create some utility which wraps that and export that instead. It's a little unfortunate that people have to do that manually.